docs/actuator-api-spec/backup-management-api.yaml

```diff
@@ -147,20 +204,6 @@ components:
           type: string
           format: date-time
           example: "2022-09-15T13:10:38.176514094Z"
-        descriptor:
-          readOnly: true
```

Comment:
:thought_balloon: the `PartitionBackupInfo` component is also used as the `list` response type where we will not include all fields. Now they are not marked as required, which means it's valid to omit them from the list result. 
IMO we should consider adding a second component type that is only used in list result. I think that would be a bit clearer.

Reply:
If we do that, then there is a chance that we accidentally deviates the spec of list response from the get response.

I think it is ok to keep it like this. The details are only for humans anyway. May be later we also decide to include full details in the list response.